.github/workflows/release.yaml

```diff
@@ -1,52 +1,84 @@
 name: Release
 
```

Comment:
```suggestion
run-name: release ${{ github.event.inputs.tag }}
```

Reply:
would this keep the release workflow name the same?

Reply:
yes, here is an example: https://github.com/scality/vault2/actions/workflows/release.yaml

Reply:
pretty cool, thanks